### DISCUSSION TITLE
Order of components in a reaction

### DISCUSSION THREAD

**Discussion by ivanderbenev** [jsme-editor/jsme-editor.github.io#56] [Q&A] [answered]
Hello,
when I use the JSME in the reaction mode, it exports the reaction components in the reaction SMILES based on the order I draw the components, and not in the order of where they really are in the editor. Is there any way to change that behaviour to export the components based on their place? Thanks!

**Comment by peter-ertl** [comment, marked as answer]
Hello Ivan,
this feature is currently not possible. Hopefully you can live with the
order as reactants as provided by JSME.

**Reply by ivanderbenev** [reply]
Thanks, Peter!
